src/main/kotlin/christmas/view/InputView.kt

```diff
@@ -0,0 +1,51 @@
+package christmas.view
+
+import camp.nextstep.edu.missionutils.Console
+import christmas.domain.order.Menu
+import christmas.domain.order.Order
+import java.time.LocalDate
+
+object InputView {
+    fun readOrder(): Order {
+        println("Hello! This is the December Event Planner for the Woowa Restaurant.")
+        return readMenu(readVisitDate())
+    }
+
+    private fun readVisitDate(): LocalDate {
+        println("When do you plan to visit the restaurant in December? (Please enter numbers only!)")
+        retry {
+            val day = Console.readLine().toInt()
+            require(day in 1..31) { "Invalid day. Please enter the day again." }
+            return LocalDate.of(2023, 12, day)
+        }
+    }
+
+    private fun readMenu(placedDate: LocalDate): Order {
+        retry {
+            println(
+                "Please tell us the menu items you’d like to order and their quantities.\n" +
+                        "(e.g. Seafood Pasta-2,Red Wine-1,Chocolate Cake-1)"
+            )
+            val orderItems = Console.readLine().split(",")
+                .associate { it ->
+                    val split = it.split("-")
+                    val menu = Menu.from(split[0])
+                    val quantity = split[1].toInt()
+                    menu to quantity
+                }
+            return Order.of(placedDate, orderItems)
```

Comment:
[Suggestion]

1. To make the result of `split("-")` more explicit and avoid unexpected splitting (e.g., when a menu name contains `-`), it can be better to add `limit = 2` to the `split` call.
2. Separating the parsing (`split`) and object mapping (`associate`) steps can help improve readability and make it easier to locate the exact step where an error occurs.

```suggestion
            val orderItems = Console.readLine()
                .split(",")
                .map { it.split("-", limit = 2) }
                .associate { (name, quantity) -> Menu.from(name) to quantity.toInt() }
            return Order.of(placedDate, orderItems)
```

Reply:
That's a good point. And I didn't know there's such an option. I'll apply these suggestions right away 🚀